hasher_test.go

```diff
@@ -277,6 +314,9 @@ func TestValidateSiblingsNamespaceOrder(t *testing.T) {
 }
 
 func TestValidateNodeFormat(t *testing.T) {
```

Comment:
[question] should a test case be added that is > the expected non-leaf node size?

Reply:
Sure, done in https://github.com/celestiaorg/nmt/pull/175/commits/f5427e5c71271735866d20f093c25a5689aef6b2